spec/spec_helper.rb

```diff
@@ -55,8 +55,11 @@
   config.run_all_when_everything_filtered = true
 
   # only run aws tests from CI (or w/ `--tag aws`) and only run it on the main repo, since that
-  # is where the valid aws keys live
-  config.filter_run_excluding(aws: true) unless ENV['CI'] && ENV['TRAVIS_PULL_REQUEST_SLUG'].match('samvera-labs/hyku')
+  # is where the valid aws keys live. TRAVIS_PULL_REQUEST_SLUG is "" when the job is a push job
+  unless ENV['CI'] &&
+         (ENV['TRAVIS_PULL_REQUEST_SLUG'].match('samvera-labs/hyku') || ENV['TRAVIS_PULL_REQUEST_SLUG'].blank?)
```

Comment:
In this case, could you just remove the `TRAVIS_PULL_REQUEST_SLUG` entirely?  I'm not understanding what situation this is trying to prevent. Maybe you could modify the commit message to explain a bit more.

Reply:
@jcoyne here is the description from part 1.  The issue is that only the travis ci from this repo is set to run the S3 specs successfully. So all forked pull request spec sets fail on the S3 connection tests.  BUT we want those test to run after a branch is merged to make sure there is no regression. This commit prevents the tests from running locally (as it was before) and from running on forks, but makes sure they run on pull requests from within this repo and on merged branches. This is part 2, simply because I can't test that the specs actually run like they are supposed to until it is merged it is a little tricky to know if we got the variables right for all the cases w/o getting a merge =-)

---
Fixes #1524

No longer run S3 related specs on non-samvera-labs repos. This will allow specs to pass on forks.

This is a first swing at getting Travis to not run the specs unless the code is in samvera-labs/hyku. Specs on forks will thus pass and the 4 failing S3 specs from forks will get run properly once code is merged.

Note - need to check that the specs in question are run after this is merged.
---

Reply:
Sounds good. 👍